src/Pages/Home/Home.jsx

```diff
@@ -8,7 +8,11 @@ const Home = () => {
   useEffect(() => {
     fetch("/articles/getarticles.json")
       .then((res) => res.json())
-      .then((jres) => setmsg(jres?.msg));
+      .then((jres) => {
+        if (jres.status === 200) setmsg(jres?.msg);
+        else console.log(jres);
+      })
+      .catch((e) => console.log(e));
```

Comment:
Handle error properly by creating an state object for error. And displaying the error in the DOM in case of any error.

Refer to this article:

https://medium.com/codex/handling-loading-and-error-state-in-react-application-and-clean-state-mess-b3cbf28029fd